### PR TITLE
python community summit

### PR DESCRIPTION
**Pull request by Leon12-aaryson**
- used tables to handle the schedule for community summit

### CODE REVIEW

File: app/2024/community-summit/page.jsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                  </div>
+                </td>
+                <td class="rounded shadow bg-white">
+                  <div class="p-2 text-black">
+                    <h4>Welcome remarks and Introductions</h4>
+                  </div>
+                </td>
+                <td class="rounded shadow bg-white">
+                  <div class="p-2 text-black">
+                    <h4>Jerry Kakoza</h4>
```
Comment (HassanBahati) [inline app/2024/community-summit/page.jsx:48]: `Jerry Kakooza` is apparently not available. there is a lady called `Deborah` taking up the place

Comment (Leon12-aaryson) [reply app/2024/community-summit/page.jsx:48]: I will make the necesary changes.

Comment (HassanBahati) [reply app/2024/community-summit/page.jsx:48]: okay

**Comment by Leon12-aaryson** [comment]
hey @HassanBahati please review this and recommend any changes that may be needed.

**Review by HassanBahati — APPROVED** [review]
LGTM! thank you @Leon12-aaryson :clap:

**Pull request merged**